features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/ApiConsole/PatchedSwaggerUIReact.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
```

Comment:
License header is missing

Reply:
I'm not sure whether we could add this with our lisence header , Because I have took this from 
[`swagger-ui-react`](https://github.com/swagger-api/swagger-ui/blob/master/flavors/swagger-ui-react/index.js) repo.
@TanyaM  WDYT ?